build.xml

```diff
@@ -31,86 +31,44 @@
 
-  <path id="adaptorlib.build.classpath">
+  <path id="build.classpath">
     <fileset dir="${lib.dir}">
-      <include name="json_simple-1.1.jar"/>
       <include name="gdata-core-1.0.jar"/>
       <include name="gdata-gsa-1.0.jar"/>
-      <include name="joda-time-1.6.jar"/>
-      <include name="opensaml-2.3.2.jar"/>
-      <include name="openws-1.3.1.jar"/>
-      <include name="xmltooling-1.2.2.jar"/>
-      <include name="guava-17.0.jar"/>
-      <include name="jsr305.jar"/>
-      <include name="gson-2.0.jar"/>
-      <include name="velocity-1.5.jar"/>
-      <include name="commons-daemon-1.0.15.jar"/>
     </fileset>
+    <path refid="ivy.compile"/>
   </path>
 
-  <path id="adaptorlib.run.classpath">
-    <path refid="adaptorlib.build.classpath"/>
+  <path id="run.classpath">
+    <path refid="build.classpath"/>
     <pathelement location="${resource.dir}"/>
-    <path refid="opensaml.run.classpath"/>
-  </path>
-
-  <path id="examples.build.classpath">
-    <fileset dir="${lib.dir}">
-      <include name="opencsv-2.3.jar"/>
-    </fileset>
+    <path refid="ivy.runtime"/>
   </path>
 
-  <path id="examples.run.classpath">
-    <path refid="examples.build.classpath"/>
+  <path id="test.classpath">
+    <path refid="run.classpath"/>
+    <path refid="ivy.test"/>
   </path>
 
-  <path id="opensaml.run.classpath">
-    <fileset dir="${lib.dir}">
-      <include name="bcprov-jdk15-1.45.jar"/>
-      <!--include name="commons-codec-1.3.jar"/-->
-      <include name="commons-collections-3.1.jar"/>
-      <!--include name="commons-httpclient-3.1.jar"/-->
-      <include name="commons-lang-2.1.jar"/>
-      <!--include name="jcip-annotations-1.0.jar"/-->
-      <!--include name="jcl-over-slf4j-1.5.10.jar"/-->
-      <include name="joda-time-1.6.jar"/>
-      <!--include name="jul-to-slf4j-1.5.10.jar"/-->
-      <!--include name="log4j-over-slf4j-1.5.10.jar"/-->
-      <!--include name="not-yet-commons-ssl-0.3.9.jar"/-->
-      <include name="opensaml-2.3.2.jar"/>
-      <include name="openws-1.3.1.jar"/>
-      <include name="slf4j-api-1.5.10.jar"/>
-      <include name="velocity-1.5.jar"/>
-      <include name="xmlsec-1.4.3.jar"/>
-      <include name="xmltooling-1.2.2.jar"/>
-
-      <!-- Libraries that are supposed to be endorsed, that we aren't having
-           endorsed. -->
-      <!--include name="resolver-2.9.1.jar"/-->
-      <!--include name="serializer-2.9.1.jar"/-->
-      <!--include name="xalan-2.7.1.jar"/-->
-      <include name="xercesImpl-2.11.0.jar"/>
-      <include name="xml-apis-1.4.1.jar"/>
-
-      <!-- Additional dependencies not included with opensaml. -->
-      <include name="slf4j-jdk14-1.5.10.jar"/>
-      <include name="commons-logging-api-1.1.1.jar"/>
-    </fileset>
-  </path>
-
-  <path id="cobertura.classpath">
-    <fileset dir="${cobertura.dir}" erroronmissingdir="false">
-        <include name="cobertura.jar"/>
-        <include name="lib/**/*.jar"/>
-    </fileset>
+  <path id="misc.classpath">
```

Comment:
The new class paths are a little less granular. Is that just to make the Ivy config simpler?